Week3/homework/step4-bonus.js

```diff
@@ -4,7 +4,20 @@ const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 // Add your function here. Try and come up with a good name for this function
 
+function returnArray (arr){
```

Comment:
Can you think of a better name for this function? I know it returns an array but that hardly covers what the function does.